### PR TITLE
Fix deprecations

### PR DESCRIPTION
**Pull request by devmotion**
This PR should fix the deprecation warnings (deprecated `Val(false)` argument in `cholesky` and deprecated `LocationScale`).

### CODE REVIEW

File: test/ad/distributions.jl [hunk 1]
```diff
@@ -143,7 +143,7 @@
         DistSpec(Levy, (0.0,), 0.5),
         DistSpec(Levy, (0.0, 2.0), 0.5),
 
-        DistSpec((a, b) -> LocationScale(a, b, Normal()), (1.0, 2.0), 0.5),
+        DistSpec((a, b) -> a + b * Beta(), (1.0, 2.0), 0.5),
```
Comment (devmotion) [inline test/ad/distributions.jl:146]:
```suggestion
        DistSpec((a, b) -> a + b * Beta(), (1.0, 2.0), 2.0),
```

File: test/ad/distributions.jl [hunk 2]
```diff
@@ -143,7 +143,7 @@
         DistSpec(Levy, (0.0,), 0.5),
         DistSpec(Levy, (0.0, 2.0), 0.5),
 
-        DistSpec((a, b) -> LocationScale(a, b, Normal()), (1.0, 2.0), 0.5),
+        DistSpec((a, b) -> a + b * Beta(), (1.0, 2.0), 2.0),
```
Comment (yebai) [inline test/ad/distributions.jl:146]: Maybe consider adding a comment here to indicate it is meant to test an `AffineDistribution` distribution. The syntax `a+b*Beta()` is quite subtle, I think.

**Review by yebai — APPROVED** [review]
Many thanks, @devmotion - looks good except for one minor clarity comment below.

File: test/ad/distributions.jl (hunk 2 above)
Comment (devmotion) [reply test/ad/distributions.jl:146]:
```suggestion
        # Test AffineDistribution
        DistSpec((a, b) -> a + b * Beta(), (1.0, 2.0), 2.0),
```

**Pull request merged**